style.css

```diff
@@ -743,42 +881,606 @@ footer {
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 5px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    font-family: 'Inter' Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-top: 8%;
+  }
+
+  .About {
+    display: flex;
+    flex-direction: row;
+    width: 1440px;
+    height: 1100px;
+    left: 0;
+    top: 2900px;
+  }
+
+  .aboutme {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 1440px;
+    height: 1100px;
+    background: url(./images/illustration-about\ me@2x.png);
+    background-position: 100% 10%;
+    background-repeat: no-repeat;
+    background-size: 39%;
+  }
+
+  .part1 {
+    display: flex;
+    flex-direction: column;
+    width: 566px;
+    height: 148px;
+    font-family: 'Inter' Verdana, Geneva, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin-top: -22%;
+    margin-left: 10%;
+  }
+
+  .part1 h1 {
+    font-size: 72px;
+    text-align: left;
+    margin-left: 2%;
+  }
+
+  .part1 p {
+    font-weight: 400;
+    font-size: 20px;
+    text-align: left;
+    line-height: 32px;
+    font-family: 'Inter' Verdana, Geneva, Tahoma, sans-serif;
+  }
+
+  .aboutme hr {
+    margin-top: -10%;
+    border: 1px solid #fff;
+  }
+
+  .LANGUAGE {
+    display: flex;
+    flex-direction: row;
+    margin-top: -30%;
+    justify-content: space-evenly;
+    font-size: 22px;
+    font-family: 'Inter' Verdana, Geneva, Tahoma, sans-serif;
+    margin-left: -2%;
+  }
+
+  .heading {
+    font-size: 32px;
+    font-family: 'Inter' Verdana, Geneva, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 44px;
+  }
+
+  footer {
+    width: 1440px;
+    display: flex;
+    margin-top: -50%;
+    justify-content: space-between;
+  }
+
+  .footer-p {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .form-p p {
+    width: 720px;
+    font-size: 40px;
+    text-align: left;
+    margin-left: 35%;
+    color: #262626;
+    margin-top: 5%;
+    font-family: 'Inter' Verdana, Geneva, Tahoma, sans-serif;
+  }
+
+  .form {
+    width: 720px;
+  }
+
+  .icons {
+    margin-left: 8%;
+    margin-top: 5%;
+    order: 2;
+    display: flex;
+  }
+
+  .button {
+    margin-top: -30%;
+    margin-left: -25%;
+  }
+
+  .footer-social-links {
+    display: flex;
+    width: 10%;
+  }
+
+  .list5 {
+    height: 35px;
+    width: auto;
+  }
+
+  .list5:hover {
+    background-color: orangered;
+    color: white;
+  }
+
+  .lst {
+    display: none;
+  }
+
+  .list5 a {
+    margin: 5px;
+  }
+
+  .actn-bar {
+    display: flex;
+    width: 390px;
+    margin-top: 35%;
+    justify-content: first baseline;
+  }
+
+  .actn-bar:hover {
+    background-color: #f4f5f7;
+    color: #0d151e;
+  }
+}
+*/
```

Comment:
- [ ] We understand why you would prefer to keep these lines of codes commented but this is not a good way to keep track of your previous codes, it is recommended to use git history or branches that contain different comment changes, I would STRONGLY recommend you remove these commented lines